_data/navbars/maintenance.yml

```diff
@@ -31,6 +31,20 @@ section:
     path: /maintenance/ebpf/ebpf-and-eks
   - title: Troubleshooting
     path: /maintenance/troubleshoot/troubleshoot-ebpf
+- title: calicoctl
+  path: /getting-started/clis/calicoctl/
+  section:
+  - title: Install calicoctl
+    path: /getting-started/clis/calicoctl/install
```

Comment:
Hm, wonder if we should be moving the actual docs content into the new location as well.

Reply:
Sure. I can do this. (For last CE docs reorg, I did move most of the content just to keep confusion down for authors.)

Reply:
I moved docs into /Maintenance. Then I rebased. But it showed only conflict in navbars--which I know cannot be correct. I fixed that, did git add, then continued rebase and no other conflicts. So I thought maybe semaphore would complain, but I don't see semaphore on this PR. There are clearly broken links in netlify view. ?? Should I just start a new PR?

Reply:
The PR looks good to me - I think you would expect to only see conflicts in the navbar because you're moving files to a place that didn't exist before, and so there's nothing at the destination to conflict with.

The broken links seem to be because of a missing `/` in your navbar paths - I added a comment on it